ci-operator/step-registry/sdn/conf/sdn-conf-commands.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cat >> "${SHARED_DIR}/install-config.yaml" << EOF
```

Comment:
@pecameron so we could make this a bit simpler by:
1) having sdn-conf-commands.sh run everything that sdn-conf-multi-manifest-commands.sh  does, up to the `openshiftSDNConfig:` bit.
2) the single and multi-tenant confs would then just do the `openshiftSDNConfig:` bit.

Reply:
Not sure about this. Eventually multitenant and singletenant will go away and we will be left with what we have now.